blocks/identity-block/features/header-account-action/styles.scss

```diff
@@ -16,7 +16,13 @@
   }
 }
 
-.xpmedia-subs-header--mobile-logged-out-header {
+.xpmedia-subs-header--desktop-header {
+  button svg {
+    margin: 0 0.25em 0 0.5em;
```

Comment:
was this added for the formatting on the chevron icon svg? @nschubach may have some side effects for the other svg in the logged-out state